### PR TITLE
[FIX] travis_install_nightly: Chrome dev tools updated for new JS syntax  t#60040

### PR DESCRIPTION
**Pull request by CarmenMiranda**
### [T#60040](https://www.vauxoo.com/web#id=60040&view_type=form&model=project.task)

We want to have the latest version of the chrome dev tools, used to execute the headless browser where the Javascript tests are run.

Before this commit, this version is installed:  HeadlessChrome/65.0.3325.181

Causing some errors when using new Javascript syntax, like this one:

```python
UncaughtSyntaxError: Unexpected token .
odoo.tests.common.ChromeBrowserException: UncaughtSyntaxError: Unexpected token {
```

After this commit, we will have the latest always fetched from google repositories.

This was based on a workaround implemented by Javier Vega <javier@vauxoo.com>

### CODE REVIEW

File: travis/travis_install_nightly [hunk 1]
```diff
@@ -138,6 +138,14 @@ else
     # Install websocket-client, required to run JS tests
     pip install websocket-client
 fi
+# The new syntax of JS tests requires the latest tools to run
+if [ ${VERSION%.*} -ge 13 ]; then
+    wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb # downloading updated chrome dev tools
```
Comment (luisg123v) [inline travis/travis_install_nightly:143]: This should be located just below [this line](https://github.com/Vauxoo/maintainer-quality-tools/blob/819e9b7dc378c96306858c1c44a6b5deb3115289/travis/travis_install_nightly#L139)

File: travis/travis_install_nightly [hunk 2]
```diff
@@ -138,6 +138,14 @@ else
...
+    apt update
+    apt install fonts-liberation  # chrome dev tools dependency
+    apt install libgbm1  # chrome dev tools dependency
```
Comment (luisg123v) [inline travis/travis_install_nightly:146]: - Install both commands in one line
- Comment ir superfluous, we're installing chrome along with their dependencies

File: travis/travis_install_nightly [hunk 3]
```diff
@@ -138,6 +138,14 @@ else
     # Install websocket-client, required to run JS tests
     pip install websocket-client
 fi
+# The new syntax of JS tests requires the latest tools to run
+if [ ${VERSION%.*} -ge 13 ]; then
+    wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb # downloading updated chrome dev tools
+    apt update
```
Comment (luisg123v) [inline travis/travis_install_nightly:144]: ADd `-q` parameter, to don't add unnecessary info to the log

File: travis/travis_install_nightly [hunk 4]
```diff
@@ -138,6 +138,14 @@ else
     # Install websocket-client, required to run JS tests
     pip install websocket-client
 fi
+# The new syntax of JS tests requires the latest tools to run
+if [ ${VERSION%.*} -ge 13 ]; then
```
Comment (luisg123v) [inline travis/travis_install_nightly:142]: I don't find this necessary. The only reason we're doing this here and not on the big image is that the big image can't be updated anymore without fixing first a bunch of issues. So, I consider this should be always performed, independently of the version, as we would do when updating the big image (the new image is on the way, BTW)

**Review by luisg123v — CHANGES_REQUESTED** [review]
Remove "Authored-By", it's not a standard git command and we're re-writing the code anyway.

CC @moylop260

**Review by luisg123v — CHANGES_REQUESTED** [review]
Title:

- `[FIX] travis_install_nightly: ...`
- Lowercase title's short description ([guidelines](https://www.odoo.com/documentation/15.0/contributing/development/coding_guidelines.html#commit-message-header)). You may consider this a nitpick, but all Odoo commits are that way and I even have seen Odoo people correcting that.

**Comment by moylop260** [comment]
Thanks for your PR Carmen

Could you test it with a dummy and share the link of the MR, please?

File: travis/travis_install_nightly (hunk 3 above)
Comment (CarmenMiranda) [reply travis/travis_install_nightly]: The way I added is Ok??

File: travis/travis_install_nightly (hunk 4 above)
Comment (CarmenMiranda) [reply travis/travis_install_nightly]: Done

**Comment by CarmenMiranda** [comment]
> Title:
> 
> * `[FIX] travis_install_nightly: ...`
> * Lowercase title's short description ([guidelines](https://www.odoo.com/documentation/15.0/contributing/development/coding_guidelines.html#commit-message-header)). You may consider this a nitpick, but all Odoo commits are that way and I even have seen Odoo people correcting that.

The name was fixed, I just left the uppercase in the cases like Chrome and JS because they are proper nouns

**Comment by CarmenMiranda** [comment]
> Thanks for your PR Carmen
> 
> Could you test it with a dummy and share the link of the MR, please?
> 
> Using a repository <13.0 and another one >=13.0, please?

@moylop260 
Here is the dummy in >=13.0, It was green but I click in retry to include the changes that @luisg123v asked me for
https://git.vauxoo.com/vauxoo/instance/-/merge_requests/1377

I will pass to you the dummy in <13.0 when I'm done with a meeting 🙈

**Comment by CarmenMiranda** [comment]
Hi, @luisg123v @moylop260 This is the dummy in version < 13.0

https://git.vauxoo.com/absa/absa/-/merge_requests/2498

**Review by luisg123v — APPROVED** [review]
LGTM :+1:

**Comment by luisg123v** [comment]
The related issue is affecting several projects and the proposed fix is working under several versions, so merging.

**Pull request merged**